pkg/controller/hostpathprovisioner/storagepool_test.go

```diff
@@ -252,7 +252,7 @@ func verifyDeploymentsAndPVCs(podCount, pvcCount int, cr *hppv1.HostPathProvisio
 	Expect(err).ToNot(HaveOccurred())
 	for _, pvc := range pvcList.Items {
 		foundPVCs = append(foundPVCs, pvc.Name)
-		Expect(pvc.Name).To(ContainSubstring(*cr.Spec.StoragePools[0].PVCTemplate.StorageClassName))
+		Expect(pvc.Name).To(ContainSubstring(*&cr.Spec.StoragePools[0].Name))
```

Comment:
maybe a stupid question why dereference and address at the same time `*&` ?

Reply:
Not a stupid question at all, its me being stupid. Of course this makes no sense.